string class_name

float32 cx
float32 cy
float32 width
float32 height

float32 distance
bool valid
